src/database/models.rs

```diff
@@ -20,7 +20,7 @@ pub struct Restaurant {
 #[derive(Clone, PartialEq, Queryable, Serialize)]
 pub struct SmileyReport {
     pub id: i32,
-    pub pnr: String,
+    pub res_id: i32,
```

Comment:
Hvorfor er denne ændring med i dette PR?

Reply:
fordi data'en for smiley reporter har ændret til at ref til restaurantent er en tabel id og ikke en string værdi som det var til at starte med

opdagede fejlen da jeg var ved at lave test og tænker at det skal med hurtigs muligt